pack/fhv/fhvc.json

```diff
@@ -688,6 +718,93 @@
+    "traits": "Hazard. Blight.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10729",
+    "encounter_code": "blight",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "illustrator": "Peter Polach",
+    "name": "Dessication",
+    "pack_code": "fhvc",
+    "position": 229,
+    "quantity": 2,
+    "text": "<b>Revelation</b> – Put Dessication into play next to the agenda deck.\nAs an additional cost for an investigator to play a card, they must take 1 damage.\n<b>Forced</b> – At the end of the round: Discard 1 copy of Dessication from play. (Max once per round.)",
+    "traits": "Blight.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10730",
+    "encounter_code": "refractions",
+    "encounter_position": 1,
+    "faction_code": "mythos",
+    "illustrator": "Brian Valenzuela",
+    "name": "Empyrean Brilliance",
+    "pack_code": "fhvc",
+    "position": 230,
+    "quantity": 2,
+    "text": "<b>(Day [day]) Revelation</b> – Test 2 [willpower]. Increase the difficulty of this test by 2 if you have 5 or more cards in hand. If you fail, choose and discard 2 cards from your hand.\n<b>(Night [night]) Revelation</b> – Discard 1 card at random from your hand. Test [willpower] (X). X is that card's printed cost. If you fail, take 2 horror.",
+    "traits": "Power. Colour.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10731",
+    "encounter_code": "refractions",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "flavor": "Who do you see beyond the veil?",
+    "illustrator": "Brian Valenzuela",
+    "name": "Captivating Gleam",
+    "pack_code": "fhvc",
+    "position": 231,
+    "quantity": 3,
+    "text": "Surge.\n<b>Revelation</b> – Put Captivating Gleam into play in your threat area. Limit 1 per investigator.\n<b>Forced</b> – If you have no cards in hand: Take 5 horror and discard Captivating Gleam.",
+    "traits": "Power. Colour.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10741",
+    "encounter_code": "mutations",
+    "encounter_position": 1,
+    "faction_code": "mythos",
+    "illustrator": "Carlos Palma Cruchaga",
+    "name": "Sudden Mutation",
+    "pack_code": "fhvc",
+    "position": 241,
+    "quantity": 2,
+    "text": "<b>Revelation</b> – Attach to the nearest non-[[Elite]] enemy. If you cannot, search the encounter deck and discard pile for a [[Mutated]] enemy and draw it.\n<b>Forced</b> – When attached enemy is defeated: Discard cards from the top of the encounter deck until an enemy is discarded and spawn it at this location.",
+    "traits": "Power. Colour.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10742",
+    "encounter_code": "mutations",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "flavor": "\"It is not the strongest of the species that survives, not the most intelligent that survives. It is the one that is most adaptable to change\"\n-Charles Darwin",
```

Comment:
I think we need to put spaces around the - in the quotes, or else it might turn into a small bullet, like this: `\n - Charles`

Reply:
Done!